### ISSUE TITLE
✍️ Contribution period: <Eneje_Ndidiamaka_Promise> 

### ISSUE THREAD

**Issue by enejepromise** [ersilia-os/ersilia#1061]
### Week 1 - Get to know the community

- [x] Join the communication channels
- [x] Open a GitHub issue (this one!)
- [x] Install the Ersilia Model Hub and test the simplest model
- [x] Install Docker if needed, and test another model
- [x] Write a motivation statement to work at Ersilia
- [x] Submit your first contribution to the Outreachy site

### Week 2 - Get Familiar with Machine Learning for Chemistry

- [x] Select a model from the list suggested in GitBook
- [x] Download and serve the model via the Ersilia Model Hub to ensure it works
- [x] Open a repository on your GitHub user with all the necessary files
- [x] Select and clean a dataset of 1000 molecules (example notebook 1)
- [x] Run predictions for the molecules on the selected model and evaluate the results

### Week 3 - Validate a Model in the Wild

- [x] Find a suitable dataset with sufficient experimental results
- [x] Clean and standardize the dataset
- [x] Run predictions and calculate metrics.

### Week 4 - Prepare your final application

- [ ] Submit the final application in the Outreachy website

**Comment by GemmaTuron** [comment]
Hi @enejepromise 

We are on week 3 of the contribution period. Please let us know if you plan to continue with your contribution by the next 2 days, otherwise we will close this issue so we can focus on the applicants who want to make a final application to Ersilia.

**Comment by enejepromise** [comment]
Good morning, yes I am working on it. I am done with week one and currently
working on week 2, kindly bear with me. I promise to update my progress
before end of the day.

Thank you.

**Comment by enejepromise** [comment]
### My Motivation Letter To Work at Ersilia Open Source Initiative.


37 Lumumba street,
New Haven,
Enugu.
Enugu State.
Nigeria.
enejepromise@gmail.com
08138780210
March 18th, 2024.

The Executive Director,
Ersilia Open Source Initiative.
138 Argyle Street
CAMBRIDGE
CB1 3LS
England.

Dear Sir,

My Motivation To work In Ersilia Open Source Initiative.

I am writing to express my sincere interest in securing an internship at Ersilia, as I am keen to contribute meaningfully to the mission of democratizing access to AI/ML models for biomedical research.

I am particularly inspired by Ersilia's commitment to supporting research in Low Income Countries, building data science capacity, and advocating for open science. The company's strategic focus on bridging gaps and fostering collaboration aligns perfectly with my own values of integrity, innovation, and inclusivity.
While I am at the beginning of my Journey, I have passion for learning, willingness to adapt, open to correction, techeable, accountable, takes responsbliity for my actions and have a sincere desire to contribute positively to the field. 

I am excited about the prospect of learning from your team of experts and gaining hands-on experience that will not only benefit my personal growth but also contribute meaningfully to the organization's objectives.

Thank you for considering my application. I look forward to the opportunity to discuss how my strengths and aspirations align with Ersilia's goals and how I can contribute effectively to your team.

Best regards,

Eneje Ndidiamaka Promise.

**Comment by enejepromise** [comment]
Successfully Installed Docker

**Comment by enejepromise** [comment]
Successfulled fetched first Model

**Comment by GemmaTuron** [comment]
Hi @enejepromise 

Thanks for the update, please, if you have marked as completed W2 tasks let us know here the status of those so we can provide feedback.

**Comment by GemmaTuron** [comment]
Hi @enejepromise 
I do not see your application to Outreachy though you marked it as completed. Please note the project is closed to new applicants who had not registered their first contribution yet

**Comment by enejepromise** [comment]
Good afternoon Ma, I just submitted it.

**Comment by enejepromise** [comment]
@GemmaTuron Good Afternoon Ma, I have submitted my first application on Outreachy website since Last week find attached of my task https://colab.research.google.com/drive/1n3KNgQIGuX7gG-ID3Su3Dthln8SslpT5?usp=sharing

**Comment by GemmaTuron** [comment]
Hello @enejepromise 
Please abide by the template and make sure you open a repository and upload all the necessary files and explanations there

**Comment by enejepromise** [comment]
@GemmaTuron Ok Ma, I am putting all that together, and submit as asoon as possible. Thank you

**Comment by enejepromise** [comment]
@GemmaTuron Thank you Ma for been patient with me. The reason why I submitted late is because my Laptop developed fault, I have to start the task from the beginning.

Find attached the link to my Task 1 and Task 2
https://github.com/enejepromise/Ersilia--SolTraNet

**Comment by enejepromise** [comment]
@DhanshreeA @GemmaTuron Thank you for your time. Kindly bear with me it wasn't a deliberate, I have to be awake all through the night for the past 2days to meet up.

**Comment by GemmaTuron** [comment]
Hi @enejepromise 

Please do not  overwork and take care of yourself, I'd remind you that you do not need to complete all tasks to make a final application to Ersilia.
As we are nearing the end of the contribution period, please make the following changes/suggestions in your work and then start working on the final application, do not open new tasks:

- [ ] Add a README File where you explain all the work you have done and the conclusions you are reaching, so that I can better follow the rationale
- [ ] Explain the 80 cut off on the model bias notebook, as it does not give any molecules?
- [ ] Clean up the notebooks so that they are easier to follow (avoid printing installs and the likes, you can use the %%Capture for example)
Once this is completed you are ready to prepare a final application!

**Comment by enejepromise** [comment]
@GemmaTuron Thank you so much for your kind words,  I really appreciate. I will make corrections as you pointed out and revert back.
Thank you.

**Comment by enejepromise** [comment]
@GemmaTuron Thank you for the feedback. 
Kindly find below the corrected feedback update.
1. I have updated my README.md file to point out what the project is all about and all the required files has beed uploaded.
2. The 80 cut off that was in my Model Bias was a mistake and has been corrected
3. The printing of all installations has been handled as advised.
Find attached the link to the updated repository where the feedback was corrected. [Link](https://github.com/enejepromise/Ersilia--SolTraNet)

Your feedback is highly appreciated.
